client/logmon/logmon.go

```diff
@@ -110,14 +110,18 @@ type TaskLogger struct {
 
 // IsRunning will return true as long as one rotator wrapper is still running
 func (tl *TaskLogger) IsRunning() bool {
+	var lroRunning bool
+	var lreRunning bool
+
```

Comment:
You probably can simplify the code like the following:

```go
func (tl *TaskLogger) IsRunning() bool {
	lroRunning := tl.lro != nil && tl.lro.isRunning()
	lreRunning := tl.lre != nil && tl.lre.isRunning()
	return lroRunning && lreRunning
}
```

Reply:
woops yes much cleaner